editor/src/core/workers/parser-printer/parser-printer-comments.spec.ts

```diff
@@ -1,10 +1,15 @@
 import { testPrintParsedTextFile } from '../../../components/canvas/ui-jsx.test-utils'
 import { Utils } from '../../../uuiui-deps'
-import { clearParseResultUniqueIDs, testParseCode } from './parser-printer.test-utils'
+import { forEachValue } from '../../shared/object-utils'
+import {
+  clearParseResultUniqueIDs,
+  parseThenPrint,
+  testParseCode,
+} from './parser-printer.test-utils'
 import { applyPrettier } from './prettier-utils'
 
 describe('parseCode', () => {
-  it('should parse a component with comments in front of it', () => {
+  xit('should parse a component with comments in front of it', () => {
```

Comment:
This is now disabled?

Reply:
Oops, this was unintentionally committed